<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceServeurTchat</class>
 <widget class="QWidget" name="InterfaceServeurTchat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InterfaceServeurTchat</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPort"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonQuitter">
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLancer">
       <property name="text">
        <string>Lancer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEditLogs"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>InterfaceServeurTchat</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>-127</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
